scripts/lis_hyperv_platform/main.ps1

```diff
@@ -14,36 +12,83 @@ param(
 $ErrorActionPreference = "Stop"
 
 $scriptPath = Split-Path -Parent $MyInvocation.MyCommand.Definition
-$JobPath = Split-Path -Parent $VHDPath
-
 . "$scriptPath\retrieve_ip.ps1"
+. "$scriptPath\asserts.ps1"
+
+# constants
+$KERNEL_ARTIFACTS_URL = @("{0}/hyperv-daemons_{1}_amd64.deb",
+                         "{0}/linux-headers-{1}_{1}-10.00.Custom_amd64.deb",
+                         "{0}/linux-image-{1}_{1}-10.00.Custom_amd64.deb")
+
+function Prepare-LocalEnv {
+    param(
+        [String] $SharedStoragePath,
+        [String] $JobId
+    )
+
+    $path = "/var/lib/lava/dispatcher/tmp/$JobId"
+    $remotePath = "H:\$JobId"
+    $localPath = "C:$path"
+
+    $SharedStoragePath = $SharedStoragePath.Replace("\\", "\")
+
+    net use H: $SharedStoragePath /persistent:NO 2>&1 | Out-Null
+    if ($LastExitCode) {
+        throw
+    }
+
+    Assert-PathExists $remotePath
+
+    New-Item -Path $localPath -ItemType "directory" | Out-Null
+    Copy-Item -Path "$remotePath/*" -Destination $localPath -Force -Recurse
+
+    $localVHDPath = (Get-ChildItem -Filter "ubuntu-cloud.vhdx" -Path $localPath -Recurse ).FullName
+    Assert-PathExists $localVHDPath
 
-& "$scriptPath\setup_env.ps1" $JobPath $VHDPath $UserdataPath $KernelURL $InstanceName $MkIsoFS
-if ($LastExitCode -ne 0) {
-    throw
+    $lavaToolDisk = (Get-ChildItem -Filter "lava-guest.vhdx" -Path $localPath -Recurse ).FullName
+    Assert-PathExists $lavaToolDisk
+
+    $remoteVHDPath = (Get-ChildItem -Filter "ubuntu-cloud.vhdx" -Path $remotePath -Recurse ).FullName
```

Comment:
what is this?

Reply:
this is the location where id_rsa is copied for LAVA